tools/cdk-build-tools/lib/package-info.ts

```diff
@@ -147,6 +140,7 @@ export interface CDKBuildOptions {
    * Whether the package uses Jest for tests.
    * The default is NodeUnit,
```

Comment:
We can remove this comment for the default value now. It looks a little confusing.

Reply:
Ah right, missed that. Removed